app/views/mentor_notifications_mailer/new_discussion_post.html.haml

```diff
@@ -1,5 +1,7 @@
 .content
   %p Hi there,
   %p
-    A student you are mentoring has posted a new comment on their solution to #{@solution.exercise.title} on the #{@solution.exercise.track.title} track.
+    A student you are mentoring 
+    (#{display_handle(@solution.user, @solution.user_track)})
+    has posted a new comment on their solution to #{@solution.exercise.title} on the #{@solution.exercise.track.title} track.
```

Comment:
Can we delegate these methods? `@solution.exercise_title` and `@solution.track_title`.